testing-internal/src/main/java/com/linecorp/armeria/internal/testing/TemporaryFolder.java

```diff
@@ -76,18 +70,23 @@ public void delete() throws IOException {
             return;
         }
 
-        try (Stream<Path> walk = Files.walk(root)) {
-            walk.sorted(Comparator.reverseOrder())
-                .map(Path::toFile)
-                .forEach(file -> {
-                    try {
-                        file.delete();
-                    } catch (Exception e) {
-                        logger.warn("Unexpected exception while deleting file: {}", file.getName(), e);
-                    }
-                });
-        }
-
+        deleteAll(root.toFile());
```

Comment:
Question) So will the `root` directory be deleted with the current implementation?